tests/testsuite/kubevirtresource.go

```diff
@@ -51,6 +55,29 @@ var (
 	originalKV            *v1.KubeVirt
 )
 
+// GetVMGuestByArchitecture returns the VMGuest based on the Architecture.
+func GetVMGuestByArchitecture(opts ...libvmi.Option) (*v1.VirtualMachineInstance,
```

Comment:
@EdDev @xpivarc what do you think about this function? I don't like the 3 return parameter much, but I don't see right now a better way